Contents/Code/siteRealityLovers.py

```diff
@@ -0,0 +1,121 @@
+import PAsearchSites
+import json
+import re
+
+def search(results, encodedTitle, title, searchTitle, siteNum, lang, searchByDateActor, searchDate, searchSiteID):
+    Log('******SEARCH CALLED*******')
+    if searchSiteID != 9999: siteNum = searchSiteID
+
+    url = PAsearchSites.getSearchSearchURL(siteNum)
+    post_values = {
+        "sortBy": "MOST_RELEVANT",
+        "searchQuery": searchTitle,
+        "categoryId": "null",
+        "perspective": "null",
+        "actorId": "null",
+        "offset": "0",
+        "isInitialLoad": "true",
+        "videoView": "MEDIUM",
+        "device": "DESKTOP"
+    }
+    params = json.dumps(post_values)
+    req = urllib.Request(url, data=params, headers={'content-type': 'application/json;charset=UTF-8'})
+    data = urllib.urlopen(req).read()
+    searchResults = json.loads(data)
+
+    for searchResult in searchResults['contents']:
+        releaseDate = parse(searchResult['released']).strftime('%b %d %Y')
+        curID = searchResult['videoUri'].replace('/', '_').replace('?', '!')
```

Comment:
better use `String.Encode()` and `String.Decode()`, it's more reliable

Reply:
Thanks. I'm a Python noob. I typically use Java, this seemed like a useful project to brush up on Python.